hare3/remote.go

```diff
@@ -18,35 +18,38 @@ import (
 
 type NodeService interface {
 	HareRoundTemplate(ctx context.Context, layer types.LayerID, round IterRound) (*Body, error)
-	Beacon(ctx context.Context, epoch types.EpochID) (types.Beacon, error)
 	Publish(ctx context.Context, proto string, blob []byte) error
 }
 
+type beaconService interface {
+	Beacon(ctx context.Context, epoch types.EpochID) (types.Beacon, error)
+}
```

Comment:
Do these changes also need to be applied to `hare4`?

Reply:
No, hare4 doesn't support node-split mode yet. I think the reason it was skipped is that Hare4 isn't functional and won't be enabled before it's finished.